auction-deploy/tests/test_cli.py

```diff
@@ -36,6 +36,19 @@ def deployed_auction_address(runner):
     raise ValueError(f"Could not find auction address in output: {lines}")
 
 
+@pytest.fixture()
+def whitelisted_auction_address(runner, deployed_auction_address, whitelist_file):
+    """deploy an auction with all addresses whitelisted and return it's address"""
```

Comment:
its*

Reply:
Also the comment lies, since this fixture does not deploy an auction but reuse the already deployed auction.
Since we have no clean_chain fixture, this is important.
(maybe we should start having a clean_chain fixture)

Reply:
as those fixtures are not bound to sessions it should actually always deploy a new contract for each test.

Reply:
and we have a clean chain fixture? It is in the deploy tools plugin

Reply:
And I copied the it's so will also change it there.

Reply:
The clean chain fixture is not cleaning the right chain, but yeah as long as we have no session scoped fixtures it is fine

Reply:
ah yes you are right.